docs/dev/run_the_etl.rst

```diff
@@ -11,24 +11,182 @@ just want to use already processed data.
+defined asset's underlying function. The IO Managers in PUDL read and
+write dataframes to and from databases and parquet files.
+
+**Resources:**
+
+   *Resources are objects that are shared across the implementations of
+   multiple software-defined assets and that can be plugged in
+   after defining those assets.*
+
+`Resources <https://docs.dagster.io/concepts/resources>`__ are helpful
+for sharing python objects amongst assets.
+For example, PUDL uses a Datastore resource that multiple assets use
+to pull data from PUDL's raw data archives on Zenodo.
+
+Generally, inputs to assets should either be other assets or
+python objects in Resources.
+
+**Jobs**:
+`Jobs <https://docs.dagster.io/concepts/ops-jobs-graphs/jobs>`__
+are preconfigured collections of assets, resources and IO Managers.
+Jobs are the main unit of execution in Dagster.
+
+**Definitions**:
+`Definitions  <https://docs.dagster.io/concepts/code-locations>`__
+are collections of assets, resources, IO managers and jobs that can
+be loaded into dagit and executed.
+
+There are two main Definitions in the PUDL processing pipeline:
+
+1. ``pudl.ferc_to_sqlite`` :doc:`converts the FERC Form 1 DBF/XBRL files <clone_ferc1>`
+   into a single large `SQLite <https://sqlite.org>`__ database so that the data is
+   easier to extract, and so all of the raw FERC Form 1 data is available in a modern
+   format. You must run this script before you can run ``pudl_etl``.
+2. ``pudl.etl`` coordinates the "Extract, Transform, Load" process that processes
    20+ years worth of data from the FERC Form 1 database, dozens of EIA spreadsheets,
    and the thousands of CSV files that make up the EPA CEMS hourly emissions data into
    a clean, well normalized SQLite database (for the FERC and EIA data), and an `Apache
    Parquet <https://parquet.apache.org/>`__ dataset that is partitioned by state and
    year (for the EPA CEMS).
 
+Both definitions have two preconfigured jobs:
+  - ``etl_fast`` processes one year of data
+  - ``etl_full`` processes all years of data
+
+.. _run-dagit:
+
+Running the ETL with Dagit
+--------------------------
+Dagster needs a directory to store run logs and some assets. Create a new directory
+outside of the pudl respository directory called ``dagster_home/``. Then set the
+``DAGSTER_HOME`` variable to the path of the new directory:
+
+.. code-block:: console
+
+    mamba env config vars set DAGSTER_HOME=</path/to/dagster_home/directory/>
+    mamba activate pudl-dev
+
+Once ``DAGSTER_HOME`` is set, launch Dagit by running:
+
+.. code-block::
+
+    make dagit
+
+This will launch Dagit at http://localhost:3000/. You should see
+a window that looks like this:
+
+.. image:: ../images/dagit_home.png
+  :width: 800
+  :alt: Dagit home
+
+Click the hamburger button in the upper left to view the definitions,
+assets and jobs. First, you'll need to create the raw FERC databases by
+clicking on one of the ``pudl.ferc_to_sqlite`` jobs. Then select "Launchpad"
```

Comment:
Maybe separate this first sentence from the next. It sounds like in order to see the assets and jobs you need to create the raw FERC databases, but I think what this actually means in that to _run the etl_ you first need to create the raw FERC databases....